src/devices/tuya.ts

```diff
@@ -17427,6 +17433,7 @@ export const definitions: DefinitionWithExtend[] = [
         model: "TS0726_4_gang_scene_switch",
         vendor: "Tuya",
         description: "4 gang switch with scene and backlight",
+        fromZigbee: [fz.ignore_basic_report, fzLocal.TS0726_action],
```

Comment:
Either `TS0726_action` OR

```js
            tuya.modernExtend.combineActions([
                tuya.modernExtend.dpAction({dp: 1, lookup: {scene_1: 0}}),
                tuya.modernExtend.dpAction({dp: 2, lookup: {scene_2: 0}}),
                tuya.modernExtend.dpAction({dp: 3, lookup: {scene_3: 0}}),
                tuya.modernExtend.dpAction({dp: 4, lookup: {scene_4: 0}}),
            ]),
```

should not be needed, can you check which?

Reply:
Hi @Koenkk ,

I think they are both needed.

If I understand correctly, `TS0726_action` provides convert support for `scene_${msg.endpoint.ID}`, removing it will result in no log record with scene parsed.

![image](https://github.com/user-attachments/assets/8bcb4f84-c9fc-445e-ac09-15fbf96f2711)


`combineActions` provides expose support, removing this  function will lead to no `Action` exposure. 

![image](https://github.com/user-attachments/assets/654b5fac-a8ff-483a-ba72-fa82737a95fd)
![image](https://github.com/user-attachments/assets/3c9b6211-9f24-4826-9b95-3c81458f1330)


According to my tests, removing either of them seems not correct.

Reply:
Then this is the correct solution: https://github.com/Koenkk/zigbee-herdsman-converters/pull/9472/files#r2134702436 (you can remove `tuya.modernExtend.combineActions`)